static/src/javascripts/projects/common/modules/ui/subscription-banners/guardian-weekly-banner-template.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+
+import closeCentralIcon from 'svgs/icon/close-large.svg';
+import theGuardianLogo from 'svgs/logo/the-guardian-logo.svg';
+
+const isUserLoggedIn = userLoggedIn =>
+    userLoggedIn
+        ? 'site-message--subscription-banner__sign-in--already-signed-in'
+        : '';
+
+const subscriptionBannerTemplate = (
+    subscriptionUrl: string,
+    signInUrl: string,
+    userLoggedIn: boolean
+): string => `
+<div id="js-subscription-banner-site-message" class="site-message--subscription-banner site-message--gw-banner">
+    <div class="site-message--subscription-banner__inner">
+        <h3 class="site-message--subscription-banner__title">
+           Read The Guardian in Print
+        </h3>
+
+        <div class="site-message--subscription-banner__description">
+            <p>Support The Guardian's independent journalism by subscribing
```

Comment:
I'm not sure most would notice but the apostrophe here should probably be a proper one i.e. ’  rather than '